src/main/java/org/terasology/launcher/packages/PackageManager.java

```diff
@@ -31,9 +39,16 @@
 public class PackageManager {
 
     private static final Logger logger = LoggerFactory.getLogger(PackageManager.class);
+    private static final String INSTALL_DIRECTORY = "games";
+    private static final String SOURCES_FILENAME = "sources.json";
+    private static final String DATABASE_FILENAME = "packages.db";
+    private static final String CACHE_DIRECTORY = "cache";
```

Comment:
That information was added to `LauncherManagedDirectories` and the string constants should not be duplicated here.